control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources_test.go

```diff
@@ -313,16 +313,16 @@ func TestReconcileOLM(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			g := NewWithT(t)
+			g := gomega.NewWithT(t)
```

Comment:
why? I don't think these changes are necessary at all

Reply:
I can get rid of all of them and restore back the dot import but which now start to be deprecated (and never seen as a good practice).

Reply:
Restored